README.md

```diff
@@ -90,7 +90,7 @@ If you have an existing project but aren't ready to fix all the files yet you ca
 generate a todo file:
 
 ```bash
-$ bundle exec standardrb --gen-ignore
```

Comment:
Ignorant comment for still having not pulled down the code to run it:

Does standard print a warning when it's running in "todo" mode? I'm ok with naming it "todo" only if that's the case. Come to think of it, printing a progress-to-completion whenever a todo ignore was present would be pretty great.

What I want to avoid is anyone using this as a crutch to think that they're "passing" standard when the presence of a file is actually giving them a free pass

Reply:
@searls if I understand the above correctly you would like Standard to notify the user that the todo file is being used.  Currently it does not but I like that idea.

Something like:

```
$ bundle exec standardrb

Todo file found, ignoring the following files:
- file_a.rb
- file_b.rb
```

Let me know if I misunderstood what you are asking for.  Thanks.

Reply:
Yes, something like this would be good. I might make it slightly more scary:

```
WARNING: this project is being migrated to standard gradually via `whatevertodo.yml` and is ignoring these files:
```